pingdom-checks/clinical-fhir.ping

```diff
@@ -4,51 +4,51 @@ CLINICAL_FHIR_SLACK_ID="${SLACK_VASDVP_MONITORING},${SLACK_API_ALARMS_CLINICAL_F
 # Sandbox
 #
 
-# Nginx Proxy Health
+# Nginx Proxy Health (Liveness)
 pingdom save-check \
   --template https-public-200 \
-  -a name=sandbox-clinical-fhir-health \
+  -a name=sandbox-clinical-fhir-liveness \
```

Comment:
The names must remain the same. The pingdom script will update all the fields based on this field. This means that changing it will create an entirely new health-check but won't remove the old one, therefore it should not be changed.